docs/DeveloperGuide.md

```diff
@@ -164,8 +173,13 @@ The user starts by typing an add command. The example used in the diagram above
 
```

Comment:
Could add **Normal Function** and **Error Handling** subheaders? See Editcommand/searchcommand